Program.cs

```diff
@@ -614,39 +593,134 @@ public void itemShop()
+                                 new Text("(a)armour ", Colours.Protection, TypeWriter.Speed.Talk),
+                                 new Text("or ", Colours.Speech, TypeWriter.Speed.Talk),
+                                 new Text("(m)medicine ", Colours.Health, TypeWriter.Speed.Talk),
+                                 new Text(":)", Colours.Speech, TypeWriter.Speed.Talk));
+            if (hasPlayerBeenInShop == true)
+            {
+               if (playerHasSeenMorse == true)
+               {
+                   TypeWriter.WriteLine("(h) to hear it again or (l) to leave", TypeWriter.Speed.Talk);
+               }
+               else if (playerHasSeenMorse == false)
+               {
+                   TypeWriter.WriteLine("(h) to hear something cool or (l) to leave", TypeWriter.Speed.Talk);
+               }
+            }
+            else
+            {
+               TypeWriter.WriteLine("or (l) to leave", TypeWriter.Speed.Talk);  
+            }
+
             string playerItemType = Console.ReadLine();
-            
+
             switch (playerItemType.ToLower())
             {
-                case "w" : BuyWeapon(); break;
-                case "a" : BuyArmour(); break;
-                case "m" : BuyMedicine(); break;
-                case "l" : TypeWriter.WriteLine($"Bye {cottonUserName} :)",TypeWriter.Speed.Talk); break;
-                default: TypeWriter.WriteLine("Sorry but I don't understand", TypeWriter.Speed.Talk); itemShop(); break;
+                case "w": BuyWeapon(); break;
+                case "a": BuyArmour(); break;
+                case "m": BuyMedicine(); break;
+                case "l": TypeWriter.WriteLine($"Bye {cottonUserName} :)", TypeWriter.Speed.Talk); break;
+                case "h": TypeWriter.WriteLine("Sure thing");
+                morseCodeMessage();
+                shopCommanPlace();
+                break;
+
+                default: TypeWriter.WriteLine("Sorry but I don't understand", TypeWriter.Speed.Talk);
+                itemShop(); 
+                break;
             }
-            
+        }
+
+        private void somethingCool()
+        {
+            if (playerHasSeenMorse == false)
+            {
+                TypeWriter.WriteLine(new Text($"Hello {cottonUserName}, you want to hear somthing cool: yes/no"));
+                string somthingCoolAwnser = Console.ReadLine();
+
+                switch (somthingCoolAwnser)
+                {
+                    case "yes":
+                        playerHasSeenMorse = true;
+                        TypeWriter.WriteLine(new Text("Some guy came the other day and gave me this tape (make shure you can hear it :)", Colours.Speech, TypeWriter.Speed.Talk));
+                        morseCodeMessage();
+                        Thread.Sleep(1000);
+                        TypeWriter.WriteLine("strange huh?... but anyway are you looking for");  
+                        shopCommanPlace();
```

Comment:
You could simply put this call to shopCommonPlace() after you call somethingCool - that way your somethingCool methods knows nothing about what happens next .. so it is only doing 1 thing.